src/aio/aio_theme_changer.py

```diff
@@ -53,142 +65,97 @@ def __init__(
         - `dbc.Offcanvas` ("`offcanvas`")
         - `dbc.RadioItems` ("`radio`").  The themes are displayed as RadioItems inside the `dbc.Offcanvas` component.
           The `value` is a url for the theme
-        - `html.Div` is used as the `Output` of the clientside callbacks.
+        - Two `dcc.Store` used as `Input` of the clientside callbacks to provide the theme list and the assets path.
 
         The ThemeChangerAIO component updates the stylesheet  when the `value` of radio changes. (ie the user selects a new theme)
 
-        - param: `radio_props` A dictionary of properties passed into the dbc.RadioItems component. The default `value` is `dbc.themes.BOOTSTRAP`
+        - param: `radio_props` A dictionary of properties passed into the dbc.RadioItems component. The default `value` is `BOOTSTRAP`
```

Comment:
This will be a breaking change too - The value is now the link rather than the name of the of the theme.   But is this required in order to support custom stylesheets?

Reply:
Yes it was a breaking change, actually it was the opposite, it used the url instead of the name.
I preferred to use the name, but rollback in the previous commit to not have a breaking change... but forget to also rollback this description 🙏 

Corrected in the last commit.